src/libs/actions/User.js

```diff
@@ -302,6 +304,18 @@ function subscribeToUserEvents() {
 
     const pusherChannelName = `private-encrypted-user-accountID-${currentUserAccountID}${CONFIG.PUSHER.SUFFIX}`;
 
+    // Receive any relevant Onyx updates from the server
+    PusherUtils.subscribeToPrivateUserChannelEvent(Pusher.TYPE.ONYX_API_UPDATE, currentUserAccountID, (pushJSON) => {
+        SequentialQueue.getCurrentRequest().then(() => {
```

Comment:
Hey everyone 👋🏼 I know it's been a little while, but can someone explain why we needed to add `SequentialQueue.getCurrentRequest()` here? It's not clear to me what it's doing.

Reply:
The first I heard about it you were the one proposing it 😄  

https://expensify.slack.com/archives/C02HWMSMZEC/p1652731618392499?thread_ts=1652466984.820589&cid=C02HWMSMZEC

Reply:
I think I understand why this was added now, I tried to summarize it as concisely as possible [here](https://expensify.slack.com/archives/C01GTK53T8Q/p1681147855907809?thread_ts=1679595548.409089&cid=C01GTK53T8Q)

Reply:
It was always a longer goal of mine to add a `NETWORK.md` and that explanation would be perfect for it (or added as a comment here).